Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿Console.WriteLine("Введите натуральное число c");
+int c = int.Parse(Console.ReadLine());
+Console.WriteLine("Введите число x");
+double x = double.Parse(Console.ReadLine());
+double rezultat = 1;
+if (c <= 0)
+{
+    Console.WriteLine("Число n неудовлетворяет требованиям, введите другое");
+}
+else
+{
+    for(double k = 1; k<c; k ++)
+    {
+        rezultat = ((k + 1)/(k + 2)) * Math.Pow(x, k);
+    }
+}
+Console.WriteLine(Math.Round(rezultat, 2));
+
+Console.WriteLine("Введите число а, удовлетворяющее условию: 1 < a < 1.5");
+double a = double.Parse(Console.ReadLine());
+double i = 2;
+double rezultatDwa = 1.5;
+if (( a > 1)&(a < 1.5))
+{
+    while(rezultatDwa >= a)
+    {
+        Console.WriteLine(i);
+        i = i +1;
+        rezultatDwa = 1 + 1/i;
+    }
+}
+
+Console.WriteLine("Введите первое натурально число");
+int natfirst = int.Parse(Console.ReadLine());
+Console.WriteLine("Введите второе натурально число");
+int natsecond = int.Parse(Console.ReadLine());
+while (natfirst != natsecond)
+{
+    if (natfirst > natsecond)
+    {
+        natfirst = natfirst - natsecond;
+    }
+    else
+    {
+        natsecond = natsecond - natfirst;
+    }
+}
+Console.WriteLine("Наибольший общий делитель равен " + natfirst);

```

Comment:
1. Для локальных переменных используйте стиль camelCase.
2. Отделяйте смысловые части кода пустыми строками.
3. Если тело цикла или оператора условия состоит из одного оператора, то `{ }` можно не писать.
4. Используйте операцию `-=`.

```suggestion
Console.WriteLine("Введите первое натурально число");
int natFirst = int.Parse(Console.ReadLine());
Console.WriteLine("Введите второе натурально число");
int natSecond = int.Parse(Console.ReadLine());

while (natFirst != natSecond)
    if (natFirst > natSecond)
        natFirst -= natSecond;
    else
        natSecond -= natFirst;

Console.WriteLine("Наибольший общий делитель равен " + natFirst);
```